content/advanced-networking/secondary_cidr/eniconfig_crd.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Eniconfig_crd"
+date: 2019-03-02T12:47:43-05:00
+weight: 40
+---
+
+### Create ENIConfig as a CRD
+As next step, we will define a new ENIConfig custom resource definition (CRD) for your cluster. CRD's are extensions of Kubernetes API that stores collection of API objects of certain kind. In this case, we will store VPC Subnet and SecurityGroup configuration information in these CRD's so that Worker nodes can access them to configure VPC CNI plugin.
+
+Create ENIConfig.yaml using your favorite editor and paste the following
+
+```
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: eniconfigs.crd.k8s.amazonaws.com
+spec:
+  scope: Cluster
+  group: crd.k8s.amazonaws.com
+  version: v1alpha1
+  names:
+    scope: Cluster
+    plural: eniconfigs
+    singular: eniconfig
+    kind: ENIConfig
+
+```
+Apply the configuration. You may have to use --validate=false for 1.11 cluster
```

Comment:
Remove lines 7-31 since this is automatically included in clusters running CNI 1.3 AND applied when you `kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v1.3/aws-k8s-cni.yaml`

Reply:
@christopherhein I will test on newer cluster and update. Regarding other feedback on section names, I need to update sub-chapter headings with correct names. Is your concern about separate tree under Advanced networking. I created it this way anticipating we will have more tutorials that are networking focused. LMK if this is too confusing

Reply:
Not the extra nesting just the underscores in the names :)

Reply:
Gotcha! It was just an oversight. I will fix that in next commit

Reply:
these items are fixed